pro_tes/ga4gh/tes/models.py

```diff
@@ -335,26 +336,6 @@ class TesState(Enum):
     SYSTEM_ERROR = 'SYSTEM_ERROR'
     CANCELED = 'CANCELED'
 
-    @property
-    def is_finished(self):
-        """Check if a state is among the set of finished states."""
-        return self in (
-            self.COMPLETE,
-            self.EXECUTOR_ERROR,
-            self.SYSTEM_ERROR,
-            self.CANCELED,
-        )
-
-    @property
-    def is_cancelable(self):
-        """Check if a state is among the set of cancelable states."""
-        return self in (
-            self.QUEUED,
-            self.INITIALIZING,
-            self.RUNNING,
-            self.PAUSED,
-        )
-
```

Comment:
I actually kinda preferred this over the definition of just the enums in `states.py`, but it's not a big deal, both work.